addons/web_editor/static/src/js/backend/html_field.js

```diff
@@ -138,14 +139,6 @@ export class HtmlField extends Component {
                         this._qwebPlugin.sanitizeElement(this.readonlyElementRef.el);
                         // Ensure all external links are opened in a new tab.
                         retargetLinks(this.readonlyElementRef.el);
-
-                        const hasReadonlyModifiers = Boolean(this.props.record.isReadonly(this.props.fieldName));
```

Comment:
Just to make sure I understand. This used to restrict the feature to when the field was readonly. Now it's also active when it's in edit but the field isn't dirty so that it will save whether the field eventually gets focused or not (which is the bug you're fixing here). Correct?

Reply:
yes!